core-services/prow/02_config/_plugins.yaml

```diff
@@ -293,6 +293,10 @@ approve:
   repos:
   - operator-framework/operator-lib
   require_self_approval: true
+- commandHelpLink: ""
+  repos:
+  - openshift/route-monitor-operator
+  require_self_approval: false
```

Comment:
Why is this here?

Reply:
I think @georgettica generated those with the makefile